### PR TITLE
Fix link for creating an app under an organization

### LINKED ISSUE #4805: Fix url for creating an app under an organization
### What article on docs.github.com is affected?

[This'n.](https://docs.github.com/en/developers/apps/creating-a-github-app-using-url-parameters)

### What part(s) of the article would you like to see updated?

The part I'm updating in the corresponding pull request I'm about to open.

It's to fix a broken link.

### Additional information

None to declare.

### PR DESCRIPTION
**Pull request by thepeoplesbourgeois**
### Why:

The link as currently documented leads to the 404 page

**Closes #4805**

### What's being changed:

```diff
- * **Organization account:** `{% data variables.product.oauth_host_code %}/:org/settings/apps/new`
+ * **Organization account:** `{% data variables.product.oauth_host_code %}/organizations/:org/settings/apps/new`
```

### Check off the following:

yah I definitely did all of these, for sure. Twice, even.

- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
Thanks so much for opening a PR! I'll get this triaged for review ⚡

**Review by skedwards88 — APPROVED** [review]
✨ Thanks for this fix! I'll get this merged down for you.

**Comment by thepeoplesbourgeois** [comment]
Praise for the UX here: This repo's default settings after a fork prevent auto-merging on the grounds that "the base branch restricts merging to authorized users". The fact that this causes there to be a button to the right of that warning, which updates that setting and allows the auto-merge to go through only after it's clicked, gave a tremendously gratifying sense of being the person to push the changes I made "up to production".

tl;dr, thank you for letting me push the big shiny button 😸
